src/hotrod/impl/protocol/HeaderParams.cpp

```diff
@@ -82,6 +82,10 @@ uint8_t HeaderParams::toOpRespCode(uint8_t code) {
         return HotRodConstants::EXEC_RESPONSE;
     case HotRodConstants::QUERY_REQUEST:
         return HotRodConstants::QUERY_RESPONSE;
+    case HotRodConstants::AUTH_MECH_LIST_REQUEST:
+        return HotRodConstants::AUTH_MECH_LIST_RESPONSE;
+    case HotRodConstants::AUTH_REQUEST:
+        return HotRodConstants::AUTH_RESPONSE;
```

Comment:
I was actually thinking about an array, not a map, where the subscript is just an int.